test/cri-containerd/exec.go

```diff
@@ -24,3 +27,34 @@ func execRequest(t *testing.T, client runtime.RuntimeServiceClient, ctx context.
 	}
 	return response.Url
 }
+
+// execInHost executes command in the container's host.
+// `stdinBuf` is an optional parameter to specify an io.Reader that can be used as stdin for the executed program.
+// `stdoutBuf` is an optional parameter to specify an io.Writer that can be used as stdout for the executed program.
```

Comment:
should we add stderr for completeness?

Reply:
I can add it. Honestly I didn't know if we'd care about it much for tests but no harm in adding it just in case.